config/crd/bases/exp.cluster.x-k8s.io_machinepools.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.2.5
   creationTimestamp: null
-  name: machinepools.cluster.x-k8s.io
+  name: machinepools.exp.cluster.x-k8s.io
```

Comment:
Did we decide to ship the experimental types by default? 

I worry that this could cause user confusion if they see the types and then attempt to create one and it is not reconciled in any way.

I also worry that this could cause issues for downstream consumers of cluster-api that might not want to expose either subset of the experimental types or all of them, and they would potentially need to fork cluster-api or otherwise manage a way to maintain the CRD differences.

Reply:
For the first iteration of this experimental package, we talked about having one artifact including both core and experimental types. The idea is that if users want to use an experimental API type they might have to enable the feature, we'll have to make sure there is enough documentation around the feature flags in the future